examples/seismic/acoustic/operators.py

```diff
@@ -101,9 +101,9 @@ def AdjointOperator(model, source, receiver, space_order=4,
 
     v = TimeFunction(name='v', grid=model.grid, save=None,
                      time_order=2, space_order=space_order)
-    srca = PointSource(name='srca', grid=model.grid, ntime=source.nt,
+    srca = PointSource(name='srca', grid=model.grid, time_range=source.time_range,
```

Comment:
nitpicking, potentially for another PR: I see a recurrent pattern here. You could probably pass the ``PointData`` directly to the ``PointSource`` (or ``Receiver``), rather than its fields, and then fetch ``time_range, npoint, ...`` once and for all internally

Reply:
I don't follow - can you elaborate?

Reply:
multiple ``source``'s fields are used and passed to this ``PointSource``. So you could pass directly ``source``, rather than its field. In such a case, the ``PointSource``'s constructor would have to check what it's been passed (if we tolerate both options -- either a single ``PointData`` OR its fields separately). If we do that, all of the call sites become slightly smaller. Also, this scales well if, in the future, we plan to add additional fields to ``PointData`` that will also be needed by ``PointSource`` (or ``Receiver`` or whatever)

But as I said, this is just nitpicking -- you can ignore this.

To make it short, it's a stupid thing only to pass one single parameter rather than N (N=2 in this case)

Reply:
Understood